build_stage_repository

```diff
@@ -121,28 +113,30 @@ def create_modulemd(collection, version, stage_dir):
 
 
 def create_repository(repo_dir):
-    check_output(['createrepo', repo_dir])
+    check_output(['createrepo_c', repo_dir], stderr=STDOUT)
 
 
-def sync_copr_repository(collection, version, target_dir, dist, arch, source=False):
-    if source:
-        print(f"Syncing {collection} {version} Source RPM repository from Copr")
+def sync_prod_repository(collection, version, target_dir, dist, arch):
+    if arch == 'source':
+        print(f"Syncing {collection} {version} Source RPM repository from yum.theforeman.org")
     else:
-        print(f"Syncing {collection} {version} RPM repository from Copr")
+        print(f"Syncing {collection} {version} RPM repository from yum.theforeman.org")
```

Comment:
Would it make sense to determine the URL (which `prod_repository()` does and then print it here?